server/src/database/connection.js

```diff
@@ -0,0 +1,8 @@
+
+import mongoose from 'mongoose';
+
+const { DB_URL } = process.env;
+
+mongoose.connect(DB_URL, { useNewUrlParser: true, useCreateIndex: true });
```

Comment:
The connect() function also accepts a callback parameter in third argument. see: https://mongoosejs.com/docs/connections.html#callback

 You can use it to log if there are any errors connecting to database. something like: 

`(err) => if(err) console.log(err) else console.log('connected to mongodb database');`

Reply:
Also, do the logging in a file for error issues.